setup.cfg

```diff
@@ -10,14 +10,8 @@ addopts =
   --benchmark-columns=min,max,median
   -v
 filterwarnings =
-  error
-  # ignored by default
-  ignore::ImportWarning
-  ignore::DeprecationWarning
-  ignore::PendingDeprecationWarning
-  # raised by Cython, usually harmless
-  ignore:numpy.dtype size changed:RuntimeWarning
-  ignore:numpy.ufunc size changed:RuntimeWarning
```

Comment:
Can we roll this back please? The first line is what made CI failing when a warning is raised, and the rest of them makes it so we don't fail on less severe warnings.